src/Illuminate/Database/Console/Migrations/MigrateCommand.php

```diff
@@ -49,7 +49,7 @@ public function __construct(Migrator $migrator)
 	 */
 	public function fire()
 	{
-		if ( ! $this->confirmToProceed()) return;
+		if ( ! $this->confirmToProceed("About to run ".$this->name)) return;
```

Comment:
Since this is the same for every command, why not just move this to the trait? That way, you get rid of a little bit of repetition.